third-party/eazy/v1/openapi.yaml

```
@@ -1051,9 +1051,7 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/responses/emailResponse'
+                $ref: '#/components/schemas/emailsResponse'
```

```suggestion
                $ref: '#/components/schemas/emailResponse'
```
+ change from emailsResponse to emailResponse on line 2390

The idea is that other responses are in singular form (list, contactResponse, conversation, etc.)

---

Also, they should not be emails, but JIDs, IMHO.

https://api.eazy.im/v3/#blacklist-read

---

@ridlees, how do you know? The other array scheme `quickRepliesResponse` is also plural. Makes sense to me. Is there a convention (followed by tyntec) that prefers singular?

---

there is no convention sat in stone, I was just looking at other responses such as Note/Contact and conversation. Since you like plural more, ignore my suggestion :)

---

> Also, they should not be emails, but JIDs, IMHO.
> 
> https://api.eazy.im/v3/#blacklist-read
> 
> So, that could change from emailResponse to blacklistResponse

I don't know. JIDs and emails supposedly have the same format. As for now, the entity is named after what it contains (and not after what it is used for) which allows it to be reused in the future for different purpose. Both make sense to me. I this case, I prefer not to change something that works and is somehow understandable but feel free to insist on your change. I will change it if you insist.

---

The do have a same format but are not the same thing. I don't insist, but would suggest asking this on the main repo, so we are sure here.

---

OK, asked: https://github.com/tyntec/api-collection/pull/103#discussion_r827411608